All EIPIP Meetings/Meeting 012.md

```diff
@@ -0,0 +1,364 @@
+
+(Pooja) - So this thing was discussed inthe previous meeting saying that it makes sense that there should be a group that would be proactive in an emergency situation that might come up during a network upgrade or otherwise; so we picked that up from there although I know that cat hearders have already been doing that. I listed it in the agenda this time, adn if people would like to see it again we can bring it back. As yo umentioned, maybe in teh form of and EIP or something else btu if not then I'm happy to keep it on the agenda and continue the discussion with teh cat hearders.
+
+(James) - I would have to think more about it being an EIP or not, but it would fit as an informational EIP.
+
+(Pooja) - My concern here is that this emergency com thing and this document and the checklist that we are tlaking about; I am a little conservative about sharing it with too many people. This document should be with a group of people whom are actively taking responsibility of it and maybe reaching out to the stakeholders at the time of emergency. Even in an informational EIP it would be quite public.
+
+(Alita) - I think there should be an avenue to initiate or get in contact with those who can get more.
+
+(James) - As long as we're not listing out more people in teh informational EIP, like if you document the process, and maybe shephard a few people whom want ot know more about it.
+
+(William) - In other words the EIP should be process; a contact list shouldn't be part of an EIP, especially because that's not something to be set in stone and would mutate with time. So what we would be looking for in an EIP would be some sort of codefication of process but not the specific contacts. It would probably suffice to say major stakeholder or something vague like that instead of providiing contact information
+
+(Pooja) - It would make sense to create a checklist kind of thing at the time said situation emerges. And maybe adding some way to contact people whom are actively involved with the script.
+
+(William) - So it sounds like this should probably be pulled off of the EIPIP agenda and then maybe at such a time when we're ready to submit an EIP it would maybe come back again as an item. Does that sound about right?
+
+(Pooja) - Sounds good to me
+
+(James) - I'd also ask some editors about what they think; I'd recommend that, and everything else sounds good.
+
+## 5.1. Decisions
+- *12.5* - Remove emergency EIP process item until such time that it becomes pertinent
+
+# 6. Various Final Topics
+
+Video | [56:08](https://youtu.be/5PSQDT1nr2E)
+
+## 6.1. EIPIP project board
+ 
+(Pooja) - Okay, so we are almost at time but I want to share a couple of things here. We have created this EIP project code, the link is available in the agenda. Everyone can go ahead and look adn check that out. We'd love to have feedback.

```

Comment:
replace with "Okay, so we are almost at the time but I want to share a couple of things here. We have created this EIP project board, the link is available on the agenda. Everyone can go ahead and look and check that out. We'd love to hear feedback.